Sources/MeiliSearch/Model/TaskInfo.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+
+/**
+ `TaskInfo` instances represent the current transaction status, use the `taskUid` value to
+  verify the status of your transaction.
+ */
+public struct TaskInfo: Codable, Equatable {
+
+  // MARK: Properties
+
+  /// Unique ID for the current `TaskInfo`.
+  public let uid: Int
+
+  /// Unique ID for the current `TaskInfo`.
+  public let indexUid: String?
+
+  /// Returns if the task has been successful or not.
+  public let status: Task.Status
+
+  /// Type of the task.
+  public let type: String
+
+  /// Date when the task has been enqueued.
+  public let enqueuedAt: String
+
+  static func == (lhs: TaskInfo, rhs: Task) -> Bool {
+    lhs.uid == rhs.uid
+  }
+
+  public enum CodingKeys: String, CodingKey {
+    case uid = "taskUid", indexUid, status, type, enqueuedAt
```

Comment:
Same here for the multiple lines. I think it is more readable but it is up to you

Reply:
If we can adopt the raywenderlich style guide, this decision should be covered: https://github.com/raywenderlich/swift-style-guide#example-definition (in this case since the keys share the same context is ok to keep them in one line :))